### DISCUSSION TITLE
How can I extract all files from a mitmdump output file?

### DISCUSSION THREAD

**Discussion by mariomadproductions** [mitmproxy/mitmproxy#6213] [Q&A] [answered]
I want to grep through a mitmdump file, but if it do that now, its hard to tell which files the strings are located in. Is there a way to extract the mitmdump output file, or something like that?

**Comment by Prinzhorn** [comment, marked as answer]
https://github.com/mitmproxy/mitmproxy/blob/47021baaa1ef3d445e1ab0d0c24a58425ec069a4/examples/addons/io-read-saved-flows.py

**Reply by mariomadproductions** [reply]
Thanks
